File: 0001 Code Chef/0009 Circket World Cup Qualifier/implementation/0009 dart/circket_world_cup.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  int? score = int.tryParse(stringInput!);
  if(score != null){
    if(score >= 12){
      print("YES");
    }else{
      print("NO");
    }
  }
}

File: 0001 Code Chef/0009 Circket World Cup Qualifier/implementation/0010 nodejs/circket_world_cup.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});

rl.question('',(input)=>{
    score = parseInt(input);
    if(score >= 12){
        console.log("YES");
    }else{
        console.log("NO");
    }
    rl.close();
});